loki/base/lokistacks/lokistack.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   name: lokistack
   namespace: openshift-operators-redhat
 spec:
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 10000000
+        ingestionRate: 10000000
```

Comment:
if the units are in MB then is this 10TB, do we really need that much space?

Reply:
@joachimweyl Thanks for being more careful than I was checking if it was byes or mb. I have updated it to: 
```yaml
  limits:
    global:
      ingestion:
        ingestionBurstSize: 10
        ingestionRate: 10
```